Rocket.Chat/UIPickerView/CompoundPickerViewDelegate.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CompoundPickerViewDelegate.swift
+//  Rocket.Chat
+//
+//  Created by Vadym Brusko on 10/12/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+class CompoundPickerViewDelegate {
+    private var pickerHandlers: [PickerViewDelegate] = [PickerViewDelegate]()
+
+    func append(_ handler: PickerViewDelegate) {
+        pickerHandlers.append(handler)
+    }
+}
```

Comment:
Interesting... why do you think we would need more than one handler to the delegate?